components/mpas-ocean/src/shared/mpas_ocn_time_average_coupled.F

```diff
@@ -285,12 +299,15 @@ subroutine ocn_time_average_coupled_accumulate(statePool, forcingPool, timeLevel
                                                     removedIceRunoffFlux, avgRemovedIceRunoffFlux, &
                                                     avgRemovedIceRunoffHeatFlux, &
                                                     avgThermalForcingAtCritDepth
+        real (kind=RKIND), dimension(:,:), pointer :: avgThermalForcingAtZLevels
+        real (kind=RKIND), dimension(:,:), pointer :: avgThermalForcingAtZLevelsMask
+        real (kind=RKIND), dimension(:), pointer :: glcZLevels
 
         type (mpas_pool_type), pointer :: tracersPool
 
         real (kind=RKIND), dimension(:,:,:), pointer :: activeTracers
         integer, pointer :: indexTemperature, indexSalinity
-        integer :: iLevelCritDepth, iLevel
+        integer :: iLevelCritDepth, iLevel, iLevelGlc
```

Comment:
Maybe rename `iLevelCritDepth` as used in the 3d code, as it makes less sense there.